app/Bus/Commands/ComponentGroup/UpdateComponentGroupCommand.php

```diff
@@ -49,31 +49,49 @@
     public $collapsed;
 
     /**
+     * To whom should the component group be visible?
+     *
+     * @var int
+     */
+    public $visible;
+
+    /**
+     * The id of the creator of the component group.
```

Comment:
Yes, `@var int`, since I'm storing in it the id of the authenticated user.

> Also, is there a reason why this couldn't be the currently authenticated user?

What do you mean? It holds the id of the current authenticated user.